### ISSUE TITLE
tf.losses.sigmoid_cross_entropy

### ISSUE THREAD

**Issue by nooralahzadeh** [tensorflow/tensorflow#15846]
Hi,
it seems that there is a mistake in the api doc:
logits: Float [batch_size, num_classes] logits outputs of the network. 
however it should be:
logits: Float [batch_size, num_classes] Unscaled log probabilities.
since it said it is going to "Creates a cross-entropy loss using tf.nn.sigmoid_cross_entropy_with_logits"

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by poxvoculi** [comment]
@drpngx Is the 'logits' argument not actually logits?

**Comment by drpngx** [comment]
I think it's fine in that case since the labels are binary. it's individual log odds ratios.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by poxvoculi** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by rjurney** [comment]
This is very confusing. What does the loss function want?
